src/extensions/extension-renderer-api.ts

```diff
@@ -6,6 +6,8 @@ export * from "./lens-extension"
 export * from "./lens-renderer-extension"
 export { DynamicPageType, PageRegistry } from "./page-registry"
 export { AppPreferenceRegistry } from "./app-preference-registry"
+export { appEventBus, AppEvent } from "../common/event-bus"
```

Comment:
I guess `appEventBus` should not be exported via `@lens/extensions` API since it's runtime dependency.
Now it works because webpack exports everything to global runtime, wdyt?

Reply:
We are already exporting react/mobx/stores which are runtime dependencies. What's the downside here?

Reply:
Currently all imports from `@lens/extensions` looks like external package, but in fact it's runtime dependency managed by webpack.
If/when at some point we move it to `npm`/external dependency then it stops becoming "live" (now same instance used in `Lens.app` at runtime) and stops participating in app's lifecycle.

Reply:
As discussed (out of band) this should not be an issue.